src/models/actions.js

```diff
@@ -74,16 +74,47 @@ class Actions extends EventEmitter {
   }
 
   /**
+   * Get only the actions which are not associated with a specific thing
```

Comment:
Nit: Can we make this clearer that "not associated with a specific thing" actually means "associated with the gateway itself"?